x-pack/plugin/esql/qa/testFixtures/src/main/resources/k8s-timeseries-clamp.csv-spec

```diff
@@ -0,0 +1,77 @@
+clamp_of_double_no_grouping
+required_capability: ts_command_v0
+required_capability: clamp_functions
+// tag::clamp-min[]
+TS k8s
```

Comment:
I think these functions are general and don't require time-series, can you make tests with FROM with the employees dataset instead?